test.sh

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env bash
 
-function finish {
```

Comment:
I think this needs to be at the top of the doc - it should be restored here before merging

Reply:
👍